### DISCUSSION TITLE
Draw a hollow rectangle

### DISCUSSION THREAD

**Discussion by jueqing1015** [dlemstra/Magick.NET#883] [Help]
I want to draw a rectangle without filling. What should I do if I only set the border color

 // Drawables draw = new Drawables();
        // draw.Rectangle(box.x, box.y, box.x + box.w, box.y + box.h);
         //draw.StrokeColor(MagickColors.Red);
         //draw.StrokeWidth(5);
         //draw.FillColor(MagickColors.White);

**Comment by dlemstra** [comment]
You should change the `FillColor` to `MagickColors.Transparant`.
